### PR TITLE
feat(website): Attack Surface Management and Neo4j Guide

### PR DESCRIPTION
**Pull request by jsonpr**
A How-To-Guide on using CloudQuery and Neo4j as a destination for Graph Visualizations of resources and Attack Surface Management.

### CODE REVIEW

File: website/pages/how-to-guides/attack-surface-management-with-graph.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+<HowToGuideHeader/>
+
+In this guide, we will demonstrate how to set up CloudQuery for customizable Attack Surface Management (ASM) and how to get started with utilizing graph visualization to execute security queries.  With cloud infrastructure and security, graph visualization can be used effectively to understand relationships between assets, attack paths, and attack surface to aid with Attack Surface Management and improving security posture of your organization.
+
+## Walkthrough
+
+### Prerequisites
+
+In this guide, we will use Neo4j as a destination and AWS as a source.  For more information on how to set those up, see our documentation on [Neo4j](https://www.cloudquery.io/docs/plugins/destinations/neo4j/overview) and [AWS](https://www.cloudquery.io/docs/plugins/sources/aws/overview).
```
Comment (erezrokah) [inline website/pages/how-to-guides/attack-surface-management-with-graph.md:19]:
```suggestion
In this guide, we will use Neo4j as a destination and AWS as a source.  For more information on how to set those up, see our documentation on [Neo4j](/docs/plugins/destinations/neo4j/overview) and [AWS](/docs/plugins/sources/aws/overview).
```

We should not include the domain so the links point to the deploy previous domain

File: website/pages/how-to-guides/attack-surface-management-with-graph.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+```sql 
+MATCH
+  (a:aws_iam_user_access_keys),
+  (b:aws_iam_users)
+WHERE a.`_cq_parent_id` =  b.`_cq_id`
+CREATE (b)-[r:has_access_keys]->(a)
+RETURN type(r)
+```
+
+Next, let's create a relationship between IAM Users and IAM Groups.  In AWS, IAM Users can be members of IAM Groups and inherit their IAM policies.
```
Comment (erezrokah) [inline website/pages/how-to-guides/attack-surface-management-with-graph.md:73]:
```suggestion
Next, let's create a relationship between IAM Users and IAM Groups. In AWS, IAM Users can be members of IAM Groups and inherit their IAM policies.
```

File: website/pages/how-to-guides/attack-surface-management-with-graph.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+MATCH 
+  (igws:aws_ec2_internet_gateways), 
+  (vpcs:aws_ec2_vpcs)
+WHERE apoc.convert.fromJsonList(igws.attachments)[0]['VpcId'] = vpcs.vpc_id
+CREATE (vpcs)-[r:has_internet_gateway]->(igws)
+return type(r)
+```
+
+We've now created relationships for encryption and networking for our RDS instances.  
+In our sample environment, we have 4 RDS Instances.  The following image shows the following and their relationships:
```
Comment (erezrokah) [inline website/pages/how-to-guides/attack-surface-management-with-graph.md:219]:
```suggestion
In our sample environment, we have 4 RDS Instances. The following image shows the following and their relationships:
```

**Review by erezrokah — APPROVED** [review]
Looks good, added a few non blocking comments

File: website/pages/how-to-guides/attack-surface-management-with-graph.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+```
+
+Let's now link Security Groups to the RDS instances.
+
+```sql
+MATCH 
+  (rds_is:aws_rds_instances), 
+  (sgs:aws_ec2_security_groups) 
+WHERE 
+  apoc.convert.fromJsonList(rds_is.vpc_security_groups)[0]['VpcSecurityGroupId']
```
Comment (hermanschaaf) [inline website/pages/how-to-guides/attack-surface-management-with-graph.md:179]: Could it be a problem that this seems to be selecting only the first entry in the list?

File: website/pages/how-to-guides/attack-surface-management-with-graph.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+description: >-
+  How to use CloudQuery for Attack Surface Management (ASM) and Graph Visualization with Neo4j
+author: jsonkao
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+
+<HowToGuideHeader/>
+
+In this guide, we will demonstrate how to set up CloudQuery for customizable Attack Surface Management (ASM) and how to get started with utilizing graph visualization to execute security queries.  With cloud infrastructure and security, graph visualization can be used effectively to understand relationships between assets, attack paths, and attack surface to aid with Attack Surface Management and improving security posture of your organization.
```
Comment (hermanschaaf) [inline website/pages/how-to-guides/attack-surface-management-with-graph.md:13]: I'd suggest placing one of the screenshots at the top of the post to help grab attention and give the reader an idea of what they'll get

**Review by hermanschaaf — APPROVED** [review]
Awesome post! 🎉

File: website/pages/how-to-guides/attack-surface-management-with-graph.md (hunk 5 above)
Comment (yevgenypats) [reply website/pages/how-to-guides/attack-surface-management-with-graph.md:13]: Yeah the graph screen shots are pretty cool!

**Review by yevgenypats — APPROVED** [review]
Looks great! I don't have anything to add on top of the current comments. Thanks @hermanschaaf and @erezrokah !

**Pull request merged**